inst/pages/97_extra_materials.qmd

```diff
@@ -639,16 +639,16 @@ under altExp.
 
 Tax_glom removes the taxa which have not been assigned to the level given in
 taxrank by default (NArm = TRUE).
-So we will add the onRankOnly = TRUE to mergeFeaturesByRank function which is
+So we will add the na.rm = TRUE to agglomerateByRank function which is
 equivalent to the default behaviour of tax_glom.
 
 ```{r}
-#| label = "mergeFeaturesByRank"
+#| label = "agglomerateByRank"
 
 altExp(tse, "Family") <-
-  mergeFeaturesByRank(tse,
+  agglomerateByRank(tse,
                       rank = "Family",
-                      onRankOnly = TRUE, # Let's add the equivalent option to NArm =TRUE
+                      na.rm= TRUE, 
```

Comment:
na.rm= --> na.rm =

Reply:
As this is the default option, you could just remove the parameter na.rm and agglomerate.tree